memory_enhancer_app/android/app/src/main/res/values/credentials.xml

```diff
@@ -0,0 +1,9 @@
+<resources>
+    <!--IBM Speech to Text Service Credentials -->
+    <string name="speech_text_iam_apikey">b1rIMw3vSUe86N7BK7efz6cDorY-g1qTxUPkC2dmmJ6J</string>
+    <string name="speech_text_url">https://api.us-east.speech-to-text.watson.cloud.ibm.com/instances/e8365b62-16bf-48e8-8bd8-cc9b0b556e58</string>
```

Comment:
Unless this has been discussed previously and I'm not aware of it, credentials should not be committed. You can use the .gitignore file for this